examples/hpc-slurm-h4d/README.md

```diff
@@ -0,0 +1,70 @@
+# Slurm H4D Blueprint
+
+This blueprint uses the HPC Toolkit to provision a Slurm cluster on Google Cloud, specifically utilizing H4D instances for compute nodes. It sets up the necessary networking, storage (Filestore for home directories), and Slurm components (controller, login, compute nodes).
+
+H4D instances are optimized for tightly-coupled High Performance Computing (HPC) workloads, offering features like RDMA.
+
+## Prerequisites
+
+1. **Google Cloud SDK:** Ensure `gcloud` is installed and configured.
+2. **Cluster Toolkit (`gcluster`):** Install cluster toolkit. Instructions can be found at [cluster toolkit's readme](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/README.md).
```

Comment:
in this link there is no such instruction on how to install cluster toolkit.

Shouldn't this be replaced by this link https://cloud.google.com/cluster-toolkit/docs/setup/install-dependencies ?